integreat_cms/core/management/commands/send_push_notifications.py

```diff
@@ -45,9 +49,14 @@ def handle(self, *args, **options):
             draft=False,
             scheduled_send_date__lte=timezone.now(),
         )
-        for pn in pending_push_notifications:
-            self.send_push_notification(pn)
-        self.print_success("Done!")
+        if pending_push_notifications:
+            for pn in pending_push_notifications:
+                self.send_push_notification(pn)
+            logger.info("All scheduled push notifications have been processed.")
```

Comment:
Could we add a count or some representation of the sent PNs? I guess this might come in handy if the feature isn't working or used as intended :)

Reply:
@ulliholtgrave good idea, maybe like this?